test/CadlRanchProjects.Tests/payload-multipart.cs

```diff
@@ -16,7 +16,7 @@ public class PayloadMultipartTests : CadlRanchTestBase
         [Test]
         public Task Payload_Multipart_FormData_Basic() => Test(async (host) =>
         {
-            MultiPartRequest body = new MultiPartRequest("123", File.OpenRead(SampleJpgPath));
+            MultiPartRequest body = new MultiPartRequest("123", System.IO.File.OpenRead(SampleJpgPath));
```

Comment:
I would prefer we add a type alias `using File = System.IO.File;` to avoid changing of all these code.
but since it has been done, it is fine if you do not want to change again.